MIP/move-bridge-architecture/README.md

```diff
@@ -0,0 +1,165 @@
+# MIP-\<number\>: MOVE Token -- Bridge Design
+
+- **Description**: Architecture of the bridge for Move token.
+- **Authors**: [Franck Cassez](mailto:franck.cassez@movementlabs.xyz)
+- **Desiderata**: [MIP-\<number\>](../MIP/mip-\<number\>)
+
+## Abstract
+
+This MIP describes the high-level architecture of the MOVE token bridge. The architecture describes the main bridge components and high-level requirements.
+
+## Motivation
+
+The Movement chain (L2) uses the \$MOVE token to pay for gas fees. As a result users need to hold \$MOVE tokens to pay for their transactions.
+The _native_ \$MOVE token is an ERC-20 contract on Ethereum (L1).  By native, we mean that this is the location where the token is minted and burned and where the total supply is set and possibly modified (inflation/deflation). The \$MOVE token reserve is in the L1 contract.
+
+To use the Movement chain and pay for gas fees, a user will acquire \$MOVE (native) tokens on L1, and bridge them to L2. On the L2 they can the token to pay for gas fees or with any other dApps that transact the \$MOVE token.
+A user can choose to migrate their L2 \$MOVE tokens back to the L1 at any time.
+Cross-chain transfers are usually done through a bridge.
+
+### A standard bridge architecture
+
+The process of transferring tokens across different chains is implemented with a _bridge_ (between the chains).
+
+There are several choices for the architecture of a bridge, and we describe here a classical bridge with a  _lock-mint_ protocol (see Chainlink's [What Is a Cross-Chain Bridge?](https://chain.link/education-hub/cross-chain-bridge) for a quick introduction to types of bridges).
+
+In the sequel, we use
+
+- **L1\$MOVE** for the native token on L1,
+- **L2\$MOVE** for the _wrapped_ token on L2.
+
+> [!WARNING]  
+> The transfer of tokens is one-to-one: a user bridges $k$ L1\$MOVE tokens to L2, and they receive $k$ L2\$MOVE tokens. Same one-to-one ratio applies from L2 to L1.  The bridge does not allow for _swapping_ tokens.
+
+The main idea of the _lock-mint_ protocol is as follows. For the sake of simplicity, assume the two chains (L1 and L2) have only one user and the user has an account `l1acc` on L1, and another account `l2acc` on L2.  We also assume that each transfer is for one token.
```

Comment:
> For the sake of simplicity, assume the two chains (L1 and L2) have only one user and the user has an account `l1acc` on L1, and another account `l2acc` on L2.

I would not place such an assumption here. How about assume there is a mapping from L2 account <> L1 account. and that the user has keys for both..

Reply:
it is assumption to explain the high-level operation of the bridge. It is there to amplify the explanation.
It is not necessary to have a map L1 accounts <> L2 accounts. In principle, the transfer allows anyone with the secret to withdraw on the other side.
And it may be the case that some accounts do not exist, with funds lost (we cannot do much about it, except using some best practices like eip-55).

Reply:
in that case how about instead of saying there are only two accounts, to say that `l2acc` can receive the locked funds on L2. the simplification of that only 1 account only exists is a bit strange

Reply:
it is not an assumption that we have in practice (not somehthing like "les than 1/3 valdiators are OK").
It is an assumption for the explanation of how the bridge works. 
It enables us to have the two accounst "fixed" and simplify the discussion. 
But tis holds for any l2acc and l2acc.